### PR TITLE
(QC-150) Allow tasks to run without DS

### PR DESCRIPTION
**Pull request by Barthelemy**
- Config file : introduction of "dataSource" whose "type" can be either "direct" or "dataSamplingPolicy".
- At the moment, one should call qcRunBasic with "--no-data-sampling" AND update the config file (see above)
- only implemented for qcRunBasic

### CODE REVIEW

**Comment by Barthelemy** [comment]
In the end, if someone wants to have a direct connection, they should use "direct" dataSource and remove all the dataSamplingPolicies related to the Dispatcher. 

runBasic has an option to switch between two different config files that illustrate the two options.

**Comment by Barthelemy** [comment]
ready for review

File: Framework/src/TaskRunner.cxx [hunk 1]
```diff
@@ -182,11 +181,37 @@ void TaskRunner::populateConfig(std::string taskName)
...
+    std::string type = dataSourceTree.get<std::string>("type");
+
+    if (type == "dataSamplingPolicy") {
+      auto policyName = dataSourceTree.get<std::string>("name");
+      LOG(INFO) << "policyName : " << policyName;
+      mInputSpecs = framework::DataSampling::InputSpecsForPolicy(config, policyName);
+    } else if (type == "direct") {
+
+      auto subSpecString = dataSourceTree.get<std::string>("subSpec");
+      LOG(INFO) << "subSpecString : " << subSpecString;
```
Comment (knopers8) [inline Framework/src/TaskRunner.cxx]: I guess this specific log is a leftover from the development. If you want it, the full InputSpec has the operator<< overloaded.

Comment (Barthelemy) [reply Framework/src/TaskRunner.cxx]: Totally, I will remove it now.

**Pull request merged**